### PR TITLE
fix(docs): Incorporate feedback from Anton

### PR DESCRIPTION
**Pull request by aquamatthias**
# Description

- not clear when docker setup is ready to go
- find expired certificates search is wrong
- show difference and prefer @utc@ over local @now@
- make clear search is case sensitive

# To-Dos

- [x] Format with `yarn format`
- [x] Lint with `yarn lint`
- [x] Build successfully with `yarn build`

# Code of Conduct

By submitting this pull request, I agree to follow the [code of conduct](https://resoto.com/code-of-conduct).

### CODE REVIEW

File: docs/getting-started/install-resoto/docker.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ $ mkdir -p resoto/dockerV2
 $ cd resoto
 $ curl -o docker-compose.yaml https://raw.githubusercontent.com/someengineering/resoto/{{latestRelease}}/docker-compose.yaml
 $ curl -o dockerV2/prometheus.yml https://raw.githubusercontent.com/someengineering/resoto/{{latestRelease}}/dockerV2/prometheus.yml
-$ docker compose up -d
+$ docker compose up
```
Comment (TheCatLady) [inline docs/getting-started/install-resoto/docker.md:44]: I disagree with the removal of `-d` here. It is not common that users will want to run `docker compose up` not in daemon mode. What is the motivation behind this change?

Comment (aquamatthias) [reply docs/getting-started/install-resoto/docker.md:44]: Motivation: people do not see when they can use Resoto and start Resotoshell.

Comment (TheCatLady) [reply docs/getting-started/install-resoto/docker.md:44]: @aquamatthias This is not the correct solution to that problem, however. We should instead provide instructions on how to verify that they can start the shell, perhaps by viewing logs?

Comment (aquamatthias) [reply docs/getting-started/install-resoto/docker.md:44]: @TheCatLady Can you share your source of information about what is common and what not?

Comment (TheCatLady) [reply docs/getting-started/install-resoto/docker.md:44]: @aquamatthias In what case will users want to `docker compose up` _not_ in daemon mode? It is hard to imagine that a user would want this running in the foreground, which requires keeping the terminal open...

Popular publishers of Docker images like LinuxServer.io also instruct users to execute `docker compose up -d`: https://docs.linuxserver.io/general/docker-compose

Comment (TheCatLady) [reply docs/getting-started/install-resoto/docker.md:44]:
```
docker compose logs -f <service_name>
```
would allow users to view live logs without the possibility of terminating the service if they accidentally close the terminal window.

Comment (aquamatthias) [reply docs/getting-started/install-resoto/docker.md]: my thinking: people that try it out the first time (getting started) benefit from seeing all the logs. If you put it into prod, you know how to do. On the other hand is a grep more user-friendly. Changed it to grep.

Comment (TheCatLady) [reply docs/getting-started/install-resoto/docker.md]: `grep` is definitely a more user-friendly option! And I don't know if we should make assumptions about users wanting run in daemon mode or not :)

**Review by TheCatLady — APPROVED** [review]
